test/modules/load-gen/PriorityMessageSender.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace LoadGen
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Extensions.Logging;
+
+    class PriorityMessageSender : SenderBase
+    {
+        readonly string[] outputs = new string[] { "pri0", "pri1", "pri2", "pri3" };
```

Comment:
suggestion: just define the prefix "pri" and define the upper bound of priority instead of hard-coded it on line 37.

Reply:
as per the other comment, we probably shouldn't assume sequentially incremental priority values in the testcase, since the product code doesn't make that assumption either.

---
In reply to: [397495161](https://github.com/Azure/iotedge/pull/2735#discussion_r397495161) [](ancestors = 397495161)

Reply:
Sure; then please define a set of priority in Settings class and allow it to be passing in.

Reply:
Sure; then you may need to define a list of priority in settings class and allow to pass in.